handsontable/src/plugins/dialog/__tests__/methods/hide.spec.js

```diff
@@ -0,0 +1,87 @@
+describe('Dialog - hide method', () => {
+  const id = 'testContainer';
+
+  beforeEach(function() {
+    this.$container = $(`<div id="${id}"></div>`).appendTo('body');
+  });
+
+  afterEach(function() {
+    if (this.$container) {
+      destroy();
+      this.$container.remove();
+    }
+  });
+
+  it('should hide dialog when closable is true', async() => {
+    const hot = handsontable({
+      data: [['A1', 'B1'], ['A2', 'B2']],
+      dialog: {
+        closable: true,
+      },
+    });
+
+    const dialogPlugin = hot.getPlugin('dialog');
+
+    dialogPlugin.show();
+    expect(dialogPlugin.isVisible()).toBe(true);
+
+    dialogPlugin.show();
+
+    expect(dialogPlugin.isVisible()).toBe(true);
+    dialogPlugin.hide();
```

Comment:
There is still here and there problem with formatting. May I recommend following a code formatting style where the expectation (`expect`) is separated from the test logic by a newline? Can you review all test files?